.travis.yml

```diff
@@ -15,6 +15,7 @@ install:
 - pip install .
 - pip install -r requirements.txt
 - pip install -r test-requirements.txt
+- pip freeze
```

Comment:
yes. somehow, the unit tests were locally passing but not on travis

I use the pip freeze at the end of the build to inspect which versions of the packages are installed